modules/encrypted-bucket/main.tf

```diff
@@ -1,4 +1,4 @@
-resource "aws_s3_bucket" "new_bucket" { #tfsec:ignore:AWS002
+resource "aws_s3_bucket" "new_bucket" { #tfsec:ignore:AWS002, AWS077 and AWS098
```

Comment:
Is this the right syntax for ignore?

This is the syntax they have in the example:
```
#tfsec:ignore:aws-s3-enable-bucket-encryption tfsec:ignore:aws-s3-enable-bucket-logging
resource "aws_s3_bucket" "my-bucket" {
  bucket = "foobar"
  acl    = "private"
}
```

https://aquasecurity.github.io/tfsec/v0.63.1/getting-started/configuration/ignores/

Reply:
Also could you include a comment about why they're being ignored?

Reply:
Dan, I modified the ignore syntax.
In regards to why we are ignoring them, the idea is to keep the compatibility, we can review each case afterward in a separate ticket.